src/alfasim_sdk/_internal/alfacase/alfacase_to_case.py

```diff
@@ -1492,7 +1506,9 @@ def load_annulus_description(
         "pvt_model": load_value,
         "top_node": load_value,
         "initial_conditions": load_initial_conditions_description,
-        "gas_lift_valve_equipment": load_gas_lift_valve_equipment_description,
+        "equipment": get_instance_loader(
```

Comment:
We need to update the HISTORY.rst with the leak addition and also mention this breaking change in annulus (from plain `gas_lift_valve_equipment` to hierarchy with `equipment`).